app/src/main/java/com/android/sample/ui/mlFeedback/CoachFeedbackScreen.kt

```diff
@@ -131,6 +144,33 @@ fun CoachFeedbackScreen(navigationActions: NavigationActions, cameraViewModel: C
       })
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun NoteInfoDialogue(onDismiss: () -> Unit, modifier: Modifier) {
+  BasicAlertDialog(
+      onDismissRequest = onDismiss,
+      modifier = modifier,
+  ) {
+    Card {
+      Column(modifier = Modifier.padding(16.dp)) {
+        Text(
+            text = "Note Information",
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.padding(bottom = 8.dp))
+        Text(
+            "The grades are in the following order: \n" +
+                "S\n" +
+                "A\n" +
+                "B\n" +
+                "C\n" +
+                "D\n" +
+                "S being the best and meaning you're doing it perfectly\n" +
+                "With D however, you should consider learning to do the exercise properly in order to train efficiently and not get injured")
+      }
```

Comment:
Wouldn't it be better to add this text to the strings.xml file ?

Reply:
I added it now